### ISSUE TITLE
Insufficient contrast for severe congestion text in night theme

### ISSUE THREAD

**Issue by 1ec5** [mapbox/mapbox-navigation-ios#1192]
In the night theme, the text in the bottom bar that indicates severe congestion is a dark red against a dark blue background. This is too little contrast to read on a phone screen at night.

<img src="https://user-images.githubusercontent.com/1231218/36927108-3970fcda-1e30-11e8-98e4-de4e2070b6ce.png" width="300" alt="severe">

/cc @frederoni @vincethecoder

**Comment by 1ec5** [comment]
Fixed in #1228.
